test/testdata/reference/t_multiline_file.js

```diff
@@ -0,0 +1,12 @@
+setup({
```

Comment:
nit: what's the t_ mean in the filename?

Reply:
In testTransform.js I have a comment saying "Transformed files have prefix 't_'. Should I just rename the file to be transformed_multiline_file.js?

Reply:
Yeah, let's rename if you don't mind - obvious is always better than having a comment :)